### DISCUSSION TITLE
2a w13 鋼球平衡台零組件與 NXOpen Python 程式回報區

### DISCUSSION THREAD

**Discussion by mdecycu** [mdecd2024/2astud-2asite#33] [General]
請各分組將已經完成的鋼球平衡台零組件 NXOpen Pyhon 程式與零組件檔案下載相關網址, 回報至以下留言區.

**Comment by probe2004** [comment]
第一組 零件在檔案在雲端硬碟 https://drive.google.com/drive/folders/1BFCF1uZ1_37gER7gu4ghxTeTzkzHRQ0X?usp=drive_link

**Comment by 41123134** [comment]
第三組 鋼球平衡台零組件 NXOpen Pyhon 程式與零組件檔案 
[檔案連結](https://nfuedu-my.sharepoint.com/:f:/g/personal/41123134_nfu_edu_tw/Es6l4diIBWZFvFZVu-n8V-cBAfBXGoRqM-aD44yodpB6HA?e=BEuRtb)

**Comment by 41023124** [comment]
第六組 鋼球平衡台Onedrive下載連結 https://drive.google.com/drive/folders/1bYRx5cun-KOXZZZAd22t3SKQe90JsO46?usp=drive_link
            鋼球平衡台倉儲網站 https://mdecd2024.github.io/2a-midag6/content/w12.html

**Comment by kevingg601** [comment]
第四組 零件檔案 [[](https://nfuedu-my.sharepoint.com/:f:/g/personal/41123111_nfu_edu_tw/EuSo65VZCXdBuBo8r_UuHoQBxcIrVMDLBY2hkap6bZuQ4Q?e=Z9EEuu)](https://nfuedu-my.sharepoint.com/:f:/g/personal/41123111_nfu_edu_tw/EuSo65VZCXdBuBo8r_UuHoQBxcIrVMDLBY2hkap6bZuQ4Q?e=Z9EEuu)

**Comment by Jun41123** [comment]
第七組 零件檔 https://nfuedu-my.sharepoint.com/:f:/g/personal/41123141_nfu_edu_tw/EuQU_8UaliRAimsBQZiEbB4Bqb_x16OI4V_spakyz8M5qA

**Comment by 41123130** [comment]
第二組零件(prt):https://drive.google.com/drive/folders/1DSZABrKTqbpG9xjIr65EPQa-CKXW9Xqb?usp=sharing
